### DISCUSSION TITLE
DEW_HEATER (DHT22) stop working after last update

### DISCUSSION THREAD

**Discussion by radoslawo81** [AllskyTeam/allsky-modules#53] [General]
Today i saw that my DHT stop dispalying value. First i was checked the wires and chenge the module DHT for another. Still was not working. I was reinstall all system for new. I was checkes the DHT in Python (simply test code). 
Do you have some idea? Before all works very well.

**Comment by Alex-developer** [comment]
Have you updated to the latest modules? If you have just upgraded Allsky to v2023.05.01_04 then you will also need to update the extra modules

**Reply by radoslawo81** [reply]
Thank you for fast answer. I was reinstall all: RPI system, AllSky software and modules.

**Reply by Alex-developer** [reply]
You only need to reinstall the modules, unless of course you wish to do a full rebuild

**Reply by radoslawo81** [reply]
Maybe I'm making a mistake somewhere because I'm not an advanced user. Does the address field for the DHT22 module need to be completed?

**Reply by radoslawo81** [reply]
I will try reinstall modules

**Reply by Alex-developer** [reply]
No for the DHT22 you need to set the 'Input pin' to the GPIO pin you connected the DHT22 to.

**Reply by radoslawo81** [reply]
I thought so but I wanted to make sure :)
Tomorow i will try once more time, today i was fighting with almost whole day. Thank you for help.

**Reply by radoslawo81** [reply]
Now I understand everything:). You also changed the main AllSky repository. I didn't notice this before. Now everything works. Thank you very much for help.
